css/home.css

```diff
@@ -161,6 +161,15 @@
   width: 100%;
 }
 
+.quote-container {
+  left: 0;
+  right: 0;
+  margin: auto;
+  text-align: center;
+  margin-bottom: 30px;
+  margin-top: 50px;
+}
```

Comment:
Are you sure all the properties are necessary?

Reply:
changed, now look better?

Reply:
you could use 
```
    margin-top: 50px;
    text-align: center;
```
and for Desktop
```
    margin-top: 90px;
```
Why do you need `right` and `left` properties?

Reply:
Done